dice_ml/explainer_interfaces/dice_KD.py

```diff
@@ -77,7 +77,7 @@ def _generate_counterfactuals(self, query_instance, total_CFs, desired_range=Non
         test_pred = self.predict_fn(query_instance)[0]
 
         query_instance[self.data_interface.outcome_name] = test_pred
```

Comment:
what happens if the model is a regressor? odd to return only desired_class. at least have a check.

Reply:
if the model was the regressor, then the default value of desired_class (which is "opposite") would be returned